src/moudles/controller.js

```diff
@@ -0,0 +1,22 @@
+const gameId = 'Zl4d7IVkemOTTVg2fUdzz'; //
+const baseUrl = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`;
+
+const gameScore = { user: '', score: 0 };
+
+// /////////////////// Post request
+async function postData(data = {}) {
+  const response = await fetch(baseUrl, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(data),
+  });
+  return response.json();
+}
```

Comment:
- [ ] Your leaderboard is working like a charm, great job! However, it would suggest using arrow functions as this is a better practice in Javascript, just change **every** async function's first line like this:

```
const postData = async (data = {}) => {
  // do something
}
```

Instead of:
```
async function postData(data = {}) {
  // do something
}
```
Here's a link with more [arrow functions examples.](https://stackoverflow.com/questions/42964102/syntax-for-an-async-arrow-function)